app/controllers/enterprises_controller.rb

```diff
@@ -11,19 +13,28 @@ class EnterprisesController < BaseController
 
   respond_to :js, only: :permalink_checker
 
+  def relatives
+    respond_to do |format|
+      format.json do
+        enterprises = Enterprise.find(params[:id]).andand.relatives.activated
```

Comment:
I think you might need another `andand` between `relatives` and `activated` for this to work when enterprise is not found.